packages/@aws-cdk/aws-lambda-python/test/integ.function.ts

```diff
@@ -0,0 +1,30 @@
+/// !cdk-integ cdk-integ-lambda-python
```

Comment:
This is my first experience with building and testing the CDK locally, and I had a couple of issues.

First, when running `cdk-integ` without this line, I get the following:
```
$ cdk-integ
Synthesizing integ.function.js.
Error: "cdk-integ" can only operate on apps with a single stack.

  If your app has multiple stacks, specify which stack to select by adding this to your test source:

      /// !cdk-integ STACK ...

  Available stacks: Collecting requests==2.23.0   Downloading requests-2.23.0-py2.py3-none-any.whl (58 kB) Collecting pillow==7.2.0   Downloading Pillow-7.2.0-cp36-cp36m-manylinux1_x86_64.whl (2.2 MB) Collecting certifi>=2017.4.17   Downloading certifi-2020.6.20-py2.py3-none-any.whl (156 kB) Collecting urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1   Downloading urllib3-1.25.9-py2.py3-none-any.whl (126 kB) Collecting idna<3,>=2.5   Downloading idna-2.10-py2.py3-none-any.whl (58 kB) Collecting chardet<4,>=3.0.2   Downloading chardet-3.0.4-py2.py3-none-any.whl (133 kB) Installing collected packages: certifi, urllib3, idna, chardet, requests, pillow Successfully installed certifi-2020.6.20 chardet-3.0.4 idna-2.10 pillow-7.2.0 requests-2.23.0 urllib3-1.25.9 cdk-integ-lambda-python (wildcards are also supported)
```
I haven't dug into it much, yet, but somehow I've got stdout from my Docker container leaking into the available stacks reported by `cdk-integ` 😅.

Reply:
The second issue I'm having is that the asset bundle seems to be recreated each time I run the tests, which leads to reported differences in the synth output (asset hashes are changing). This is the cause for the current build failures occurring on this branch.

Reply:
Your first issue is a bug (copy @jogold #9186).

I think python should use "SOURCE" hash type because python includes timestamps in the bundled output.